lib/code-ribbon-ribbon-container.js

```diff
@@ -33,11 +33,18 @@ class CodeRibbonRibbonContainer extends PaneAxis {
     // We always stack ribbons vertically
     this.orientation = "vertical";
 
+    if (state.patchQuota) {
+      this.patchQuota = state.patchQuota;
+    }
+    else {
+      // This will be updated later:
+      this.patchQuota = atom.config.get("code-ribbon.pane_count_calc.pane_count_horizontal_number");
```

Comment:
What would it be updated too? If we cannot find the quota, what is your thinking

Reply:
By updated later I meant it would be updated later during runtime, since this is during init.